client/src/components/settings/danger-zone.css

```
@@ -25,3 +25,129 @@
 .danger-zone p {
   color: var(--danger-color);
 }
+
+.d-flex {
+  display: flex;
+}
+
+.flex-row {
+  flex-direction: row;
+}
+
+.flex-grow-1 {
+  flex-grow: 1;
+}
+
+#dropdown-size-medium {
+  width: 100%;
+} */
+
+/* asdf */
+
+.danger-zone {
```

Looks like we can remove this comment

```suggestion
.danger-zone {
```